### DISCUSSION TITLE
Feature Request: Support for GitHub Codespaces Deployment and Development

### DISCUSSION THREAD

**Discussion by PhilippWu** [fastapi/full-stack-fastapi-template#1109] [Questions]
Hello @tiangolo ,

First of all, I want to express my appreciation for the recent updates to the project; it's been running superbly on a local environment. To further enhance the development experience, I'm wondering if it would be possible to adjust the configurations to support deployment within GitHub Codespaces. This would allow for development directly within Codespaces, streamlining the process even more.

Thank you for considering this enhancement.

Best regards

Philipp

**Comment by DiyanPabasara5** [comment]
-:)

**Comment by JonathanPlasse** [comment]
- #726
